packages/cactus-cmd-api-server/src/main/typescript/api-server.ts

```diff
@@ -732,14 +733,15 @@ export class ApiServer {
 
     this.wsApi.attach(this.httpServerApi, wsOptions);
 
-    const socketIoAuthorizer = authorizeSocket({
-      ...authzConf.socketIoJwtOptions,
-      onAuthentication: (decodedToken) => {
-        this.log.debug("Socket authorized OK: %o", decodedToken);
-      },
-    });
-
-    this.wsApi.use(socketIoAuthorizer as never);
+    if (authzProtocol != AuthorizationProtocol.NONE) {
```

Comment:
@outSH Thank you for the fix. One small nit is to swap the if condition with the same logic that the HTTP endpoints have to keep their behavior consistent in edge-cases

```suggestion
if (authorizerO.isPresent()) {
```

The presentness/emptiness of the optional there is driven by the logic within the authorizer factory which has the desired strict behavior that if you pass in some garbage value as authorization protocol it won't let you get away with it (it will throw) and therefore undefined behavior is less likely compared to us just saying that we'll carry on with whatever authorization protocol as long as it was not equal to `NONE`.

Reply:
@petermetz Done